swan-lake/integration-tutorials/transforming-message-formats.md

```diff
@@ -104,7 +104,7 @@ Follow the instructions given in this section to develop the service.
 
     ![Open diagram view](/learn/images/integration-tutorials/transforming-message-formats/open_diagram_view.gif)
 
-3. Generate record types corresponding to the payloads from the hospital service and input/output of the transforming function by providing samples of the expected JSON payloads.
+3. Generate record types corresponding to the payloads from the hospital service by providing samples of the expected JSON payloads.
```

Comment:
There is only one backend call/response/payload here?

Reply:
Yeah. Anyway I changed this to `Generate record types corresponding to the response from the hospital service by providing samples of the expected JSON payload`